spec/android_audio_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'device_api/android'
+require_relative 'helpers/stub_helper_spec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.